web/themes/custom/pineapple/assets/js/menu.behavior.js

```diff
@@ -4,17 +4,20 @@
 
     Drupal.behaviors.menu = {
       attach: function (context, settings) {
+        var win = $(window);
         var menu = $('.region-header .navigation .menu');
         $('.region-header .menu .hamburger').click(function () {
           $(menu).toggleClass('responsive');
         });
         function calculateLayout() {
-          var win = $(window);
+          var footerHeight = $('.footer').height();
+          $('.layout-container').css('padding-bottom', footerHeight);
           if (win.width() > 768 && $(menu).hasClass('responsive')) {
             $(menu).removeClass('responsive');
           }
         }
         $(window).on('resize', _.debounce(calculateLayout, 150));
+        $(document).ready(calculateLayout);
```

Comment:
You don't need to call document ready inside a behaviour, the why can be found here: https://www.drupal.org/docs/8/api/javascript-api/javascript-api-overview

You can just invoke the function directly as so:
`calculateLayout();`